### PR TITLE
1822PNW Fix coal token

### PR DESCRIPTION
**Pull request by outsidePasser**
Fixes: #8351

### CODE REVIEW

File: assets/app/view/game/corporation.rb [hunk 1]
```diff
@@ -174,6 +174,11 @@ def render_title(bg = nil)
           children << h(:img, logo_props)
         end
 
+        if @game.second_icon(@corporation)
```
Comment (tobymao) [inline assets/app/view/game/corporation.rb:177]: why is this needed?

Comment (outsidePasser) [reply assets/app/view/game/corporation.rb:177]: This was put in so that an icon can be displayed on associated minors to make it clearer which major that minor will grow up into.  It was requested in #8340, and I thought it sounded like a good idea.

**Comment by outsidePasser** [comment]
Decided to not pin - it will break a lot of games, but I don't want to chase bugs that are already fixed.
